src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.Win32.Unix.cs

```diff
@@ -49,7 +51,8 @@ internal static void CreateDirectory(string path, IsolatedStorageScope scope)
 
             DirectoryInfo info = Directory.CreateDirectory(path);
 
-            if (IsMachine(scope) && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+#if TargetsWindows
```

Comment:
But then I would have to copy the other methods in that file too. I wanted to reduce identical code...